python/lsst/pipe/base/pipeline.py

```diff
@@ -33,25 +33,10 @@
 # -------------------------------
 #  Imports of standard modules --
 # -------------------------------
+from collections.abc import Callable, Generator, Iterable, Iterator, Mapping, Set
```

Comment:
Could use `set` instead of `Set`?

Reply:
I think I did this because it was originally `AbstractSet` and ruff recommended that `Set` be used for that rather than `set` since it's the equivalent of the general `Mapping` vs `dict`.